akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -124,21 +124,33 @@ object TestPublisher {
 
     /**
      * Expect no messages.
+     * NOTE! Timeout value is automatically multiplied by timeFactor.
      */
+    @deprecated(message = "Use expectNoMessage instead", since = "2.5.5")
     def expectNoMsg(): Self = executeAfterSubscription {
       probe.expectNoMsg()
       self
     }
 
     /**
      * Expect no messages for a given duration.
+     * NOTE! Timeout value is automatically multiplied by timeFactor.
      */
+    @deprecated(message = "Use expectNoMessage instead", since = "2.5.5")
     def expectNoMsg(max: FiniteDuration): Self = executeAfterSubscription {
       probe.expectNoMsg(max)
       self
     }
 
     /**
+     * Expect no messages for a given duration.
+     */
+    def expectNoMessage(max: FiniteDuration = probe.testKitSettings.SingleExpectDefaultTimeout): Self = executeAfterSubscription {
```

Comment:
I think we should remove the default `max`, as mentioned earlier

Reply:
no problem). I just thought it is more universal).

Reply:
to clarify the rational; you "never" want this timeout be as long as the default expect timeout. That would make tests extremely slow, without adding much more safetey/coverage. You typically want this to be in the order of 100.millis. We could have another configuration property specifically for this, but making it an explicit parameter is good enough in my opinion.